init_conda_venv.sh

```diff
@@ -10,7 +10,13 @@ echo "Setting up virtualenv"
 
 # Make sure to bypass a possible $PIP_REQUIRE_VIRTUALENV=true set by users
 # This is because pip is unable to detect that it is inside conda's virtual environment - and would throw an error
-env PIP_REQUIRE_VIRTUALENV=false conda env create -f environment.yaml --prefix .venv
+env PIP_REQUIRE_VIRTUALENV=false ${CONDA_OR_MAMBA} \
```

Comment:
Question: Looks like CONDA_OR_MAMBA is not set here? Is it available indirectly from somewhere else?

Reply:
good spot! I copy and pasted only half the file